recipes/cpptrace/all/conanfile.py

```diff
@@ -21,10 +21,12 @@ class CpptraceConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "unwind_with_libunwind": [True, False],
```

Comment:
Is there any possibility to have the rest of the unwind options in the future?

We might want to have this be something like

```suggestion
        "unwind": ["libunwind", "nothing"],
```

So that in the future we can have other options without worsening the options? :)